test/spec/ol/render/canvas/index.test.js

```diff
@@ -6,7 +6,7 @@ goog.require('ol.render.canvas');
 describe('ol.render.canvas', function() {
 
   var font = document.createElement('link');
-  font.href = 'https://fonts.googleapis.com/css?family=Inconsolata';
+  font.href = 'https://fonts.googleapis.com/css?family=Abel&text=wmytzilWMYTZIL@#/&?$%10';
```

Comment:
Ithink the value of the text parameter should be url encoded.

Reply:
Good catch. Updated.